packages/api/cms-api/src/auth/strategies/auth-proxy-jwt.strategy.ts

```diff
@@ -7,7 +7,19 @@ import { CurrentUserInterface, CurrentUserLoaderInterface } from "../current-use
 
 interface AuthProxyJwtStrategyConfig {
     jwksUri: string;
-    currentUserLoader?: CurrentUserLoaderInterface;
+}
+
+@Injectable()
+export class AuthProxyJwtStrategy extends PassportStrategy(Strategy, "auth-proxy-jwt") {
```

Comment:
Is there a use case where I'd use the class directly? If yes, how would I provide the config?

If the class must always be extended, then we should make it abstract

Reply:
Done